test/unit/autocomplete.ts

```diff
@@ -226,6 +230,17 @@ suite('Autocomplete', ({ expect, spy }) => {
 
       expect(emit).to.be.calledWith('sayt:hide');
     });
+
+    it('should activate sayt when there are suggestions', () => {
+      const emit = spy();
+
```

Comment:
one thing about the test layout, I would leave no space between your `const`s and the rest of your test setup.
In an IDE `const` is enough of a visual indicator that we shouldn't additionally need a newline
This also removes ambiguity about what the "action" of your test is